rootfs-builder/rootfs.sh

```diff
@@ -52,21 +52,21 @@ $(get_distros)
 Refer the Platform-OS Compatibility Matrix: https://github.com/kata-containers/osbuilder#platform-distro-compatibility-matrix
 
 Options:
--a  : agent version DEFAULT: ${AGENT_VERSION} ENV: AGENT_VERSION 
--h  : Show this help message
--o  : specify version of osbuilder
--r  : rootfs directory DEFAULT: ${ROOTFS_DIR} ENV: ROOTFS_DIR
+-a	: agent version DEFAULT: ${AGENT_VERSION} ENV: AGENT_VERSION 
```

Comment:
Please can you revert all these whitespace changes - they are not required and make this PR very difficult to review.